app/views/components/fields/LabelledValue.scala

```diff
@@ -19,5 +19,5 @@ package views.components.fields
 case class LabelledValue(label: String, value: String)
 
 object LabelledValue {
-  def apply(labelAndValue: String): LabelledValue = LabelledValue(labelAndValue, labelAndValue)
+  def apply(labelAndValue: Any): LabelledValue = LabelledValue(labelAndValue.toString, labelAndValue.toString)
```

Comment:
Why we need to change to `Any`? Is it a possibility to receive something different than `String`?

Reply:
Yes.  To save the client calling toString on the Json object

Reply:
Can we provide two apply methods? One for `String` and second one for `JsObject`?

Using `Any` is not a good practice, you can pass everything there.